src/transformers/modeling_flax_utils.py

```diff
@@ -46,7 +46,7 @@
 
 
 ACT2FN = {
-    "gelu": nn.gelu,
+    "gelu": partial(nn.gelu, approximate=False),
```

Comment:
This is the correct gelu function for BERT -> error tolerance can be decreased to 1e-3 thanks to this correction